pkg/libmachine/load_unix.go

```diff
@@ -0,0 +1,37 @@
+// +build !windows
+
+package libmachine
+
+import (
+	"github.com/code-ready/crc/pkg/libmachine/host"
+)
+
+func (api *Client) NewHost(driverName string, driverPath string, rawDriver []byte) (*host.Host, error) {
+	driver, err := api.clientDriverFactory.NewRPCClientDriver(driverName, driverPath, rawDriver)
+	if err != nil {
+		return nil, err
+	}
+
+	return &host.Host{
+		ConfigVersion: host.Version,
+		Name:          driver.GetMachineName(),
+		Driver:        driver,
+		DriverName:    driver.DriverName(),
+		DriverPath:    driverPath,
+		RawDriver:     rawDriver,
+	}, nil
+}
+
+func (api *Client) Load(name string) (*host.Host, error) {
+	h, err := api.Filestore.Load(name)
+	if err != nil {
+		return nil, err
+	}
+
+	d, err := api.clientDriverFactory.NewRPCClientDriver(h.DriverName, h.DriverPath, h.RawDriver)
```

Comment:
was curious why this was necessary, but looking at the `_unix` and followed with a new `_windows` file it is clear to break up and prevent this on another platform where it doesn't make sense.